bubble-wrap.yaml

```diff
@@ -852,6 +878,37 @@ layers:
                 transition:
                     [show, hide]:
                         time: 0s
+                        
+    # Used by the SDK to place point, line, and polygon overlays on the map
+    mz_default_point:
+        data: { source: mz_default_point }
+        draw:
+            sdk-point-overlay:
+                interactive: true
+                sprite: search-active
+                size: [36px,54px]
+                collide: false
+                anchor: top
+                transition:
+                    [show, hide]:
+                        time: 0s
+    mz_default_line:
+        data: { source: mz_default_line }
+        draw:
+            sdk-line-overlay:
+                color: '#06a6d4'
+                order: 503
+                width: 3px
+    mz_default_polygon:
```

Comment:
Do you mean have separate polygon options for underlay or overlay? Or have the SDK set different order values on the fly (probably possible)?